data/src/main/java/org/cryptomator/data/cloud/s3/S3ClientFactory.java

```diff
@@ -45,15 +45,8 @@ private MinioClient createApiClient(S3Cloud cloud, Context context) {
 
 		MinioClient.Builder minioClientBuilder = MinioClient.builder();
 
-		if (cloud.s3Endpoint() != null) {
-			minioClientBuilder.endpoint(cloud.s3Endpoint());
-		} else {
-			minioClientBuilder.endpoint("https://s3.amazonaws.com");
-		}
-
-		if (cloud.s3Region() != null) {
-			minioClientBuilder.region(cloud.s3Region());
-		}
+		minioClientBuilder.endpoint(cloud.s3Endpoint());
+		minioClientBuilder.region(cloud.s3Region());
```

Comment:
So region will be neccessary as well?
I guess this fair.

Reply:
Yes, MinIo doesn't try to extract the region from the endpoint.